ui/app/components/ui/alert-dialog.tsx

```diff
@@ -38,34 +43,6 @@ const AlertDialogContent = React.forwardRef<
 ));
 AlertDialogContent.displayName = AlertDialogPrimitive.Content.displayName;
 
-const AlertDialogHeader = ({
```

Comment:
These are just normal divs, so no reason we de-duplicate them and re-export under a different name.